### PR TITLE
[RUN_MOBILESDK-1847] Handle 3 decimal place currencies

### PR DESCRIPTION
**Pull request by porter-stripe**
## Summary
Some currencies have 3 decimal places and it doesn't look like we were handling it anywhere.

Before this change KWD with an amount of `92000` was being presented as 920 KWD rather than 92 KWD.

## Motivation
https://jira.corp.stripe.com/browse/RUN_MOBILESDK-1847

## Testing
New unit test

## Changelog
- See diff

### CODE REVIEW

**Review by davidme-stripe — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 ## PaymentSheet
 * [Fixed] Fixed an issue that caused animations of the card logos in the Card input field to glitch.
 * [Fixed] Fixed a layout issue in the "Save my info" checkbox.
+* [Fixed] Fixed a bug where 3 decimal place currencies were not being formatted properly.
```
Comment (yuki-stripe) [inline CHANGELOG.md:5]: Is this specific to PaymentSheet?

Comment (porter-stripe) [reply CHANGELOG.md:5]: Technically no, but that's where the bug was reported. Maybe we can just put it under a `Payments` header if that makes more sense.

Comment (porter-stripe) [reply CHANGELOG.md:7]: @yuki-stripe updated the change log to put it under "Payments"

Comment (yuki-stripe) [reply CHANGELOG.md:7]: Looking at usage of this method, the bug seems to affect ~all UI surfaces that show an amount. I'm not sure what "Payments" refers to, I think we can just put at the top-level.

**Review by yuki-stripe — APPROVED** [review]

**Pull request merged**